src/rust/ide/src/view/node_editor.rs

```diff
@@ -109,6 +109,11 @@ impl<Parameter:frp::Data> FencedAction<Parameter> {
 // === GraphEditorIntegration ===
 // ==============================
 
+/// The identifier base that will be used to name the methods introduced by "collapse nodes"
+/// refactoring. Names are typically generated by taking base and appending subsequent integers,
+/// until the generated name does not collide with any known identifier.
+const COLLAPSED_FUNCTION_NAME:&str = "func";
```

Comment:
```suggestion
const COLLAPSED_FUNCTION_NAME : &str = "func";
```

Reply:
There is no spacing around `:` if it is not used to align.
And `COLLAPSED_FUNCTION_NAME` is separate from constants below (which, arguably, should not really be aligned because of comments), so we de not align here.